plugin/applescript/update.scpt

```diff
@@ -3,11 +3,12 @@ tell application "System Events"
 end tell
 
 on update_xmark()
-    tell application "{{ app }}"
+    tell application "{{ app_loc }}"
         set u to "file://{{ outurl }}"
         repeat with w in every window
             set tab_index to 0
-            repeat with t in every tab of w
+            set tablist to tabs of w
+            repeat with t in tablist
```

Comment:
For me, using the `tab` keyword seems to be the source of the syntax error, but the `tabs` keyword works. Oddly though, either seems two work. I think the key to the fix overall is the absolute path on line 6.